src/Rules/Library/LocalizedControlTypeIsNotEmpty.cs

```diff
@@ -19,13 +19,15 @@ public LocalizedControlTypeIsNotEmpty()
             this.Info.HowToFix = HowToFix.LocalizedControlTypeNotEmpty;
             this.Info.Standard = A11yCriteriaId.ObjectInformation;
             this.Info.PropertyID = PropertyType.UIA_LocalizedControlTypePropertyId;
+
```

Comment:
Remove extra newline on line 22 (you didn't add it in other classs so I'm assuming this is an oversight)